source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCardsTests/TestFiles/Action.ToggleVisibilityExhaustive.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
why was this card added but there are no changes in the test file?

Reply:
test pick up all files in the folder, and it tests rendering failure messages above.